tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1745,7 +1745,7 @@ def test_moderator_invited_moderators(self, _):
                 assert isinstance(moderator, Redditor)
 
     @mock.patch("time.sleep", return_value=None)
-    def test_modeator_leave(self, _):
+    def test_moderator_leave(self, _):
```

Comment:
Shouldn't this require renaming a cassette?

Reply:
It is renamed.

Reply:
Doh. I totally missed that 🤦